package.json

```diff
@@ -50,7 +50,8 @@
   },
   "scripts": {
     "install": "prebuild --download",
-    "test": "tape test/*-test.js | faucet"
+    "test": "tape test/*-test.js | faucet",
+    "prebuild": "rm -rf prebuilds && prebuild --verbose"
```

Comment:
`npm run prebuild --force` - this way we still have build caching which is nice when something fails